<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ucpcUDP</class>
 <widget class="QMainWindow" name="ucpcUDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ucpcUDP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblReception">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>reception</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmission">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Emission</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>RPi4.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>-1</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblInfos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>LED -&gt; GPIO 17 Bouton -&gt; GPIO18</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>163</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn0">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>LED Eteinte</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn1">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>LED Allum√©e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Etat Bouton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP serveur -&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
